src/core/test/system/users/borrower.sol

```diff
@@ -82,7 +82,7 @@ contract Borrower is DSTest {
         repayAction(loan, debt);
     }
 
-    function doApproveCurrency(address usr, uint currencyAmount) public {
-        tkn.approve(usr, currencyAmount);
+    function doApproveCurrency(address usr, uint wad) public {
```

Comment:
constantly renaming :)

Reply:
haha, no auto merge. already renamed back to currencyAmount